### PR TITLE
Webactivity changes for IR

### PR DESCRIPTION
**Pull request by mfrintu**
Adding optional "connectVia" property to the Web activity type properties.
Swagger PR: https://github.com/Azure/azure-rest-api-specs/pull/7605 
Update release notes.
Version was already changed to 4.4.0 for this release

### CODE REVIEW

**Comment by jackmagic313** [comment]
Eventually you need to use generate.ps1 to generate the C# code as well as datafactory_resource-manager.txt file once swagger change is checked-in

**Review by isra-fel — CHANGES_REQUESTED** [review]
jackmagic313 is right. Please notify me when it's done

**Comment by mfrintu** [comment]
Addressed the PR comments

File: sdk/datafactory/Microsoft.Azure.Management.DataFactory/tests/UnitTests/ActivityTests.cs [hunk 1]
```diff
@@ -171,6 +171,39 @@ public void ExecuteClientCertWebActivity_SDKSample()
...
+            Assert.Contains(activity.Authentication.Type, json);
+            JObject originalObj = JObject.Parse(json);
+            Activity restored = SafeJsonConvert.DeserializeObject<Activity>(json, client.DeserializationSettings);
+            var restoredJson = SafeJsonConvert.SerializeObject(restored, client.SerializationSettings);
+            JObject restoredObj = JObject.Parse(restoredJson);
+            Assert.True(JToken.DeepEquals(originalObj, restoredObj), string.Format(CultureInfo.InvariantCulture, "Failed at case: {0}.", triggeredPipelineName));
+        }
+		
+		[Fact]
+        public void ExecuteWebActivityOnIR_SDKSample()
```
Comment (isra-fel) [inline sdk/datafactory/Microsoft.Azure.Management.DataFactory/tests/UnitTests/ActivityTests.cs]: Need to record the test don't you?

**Review by isra-fel — CHANGES_REQUESTED** [review]
Hi @mfrintu , could you please
- Add a link to the swagger spec review PR in the description
- Update versions, if you are going to release
Thanks

File: sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -6,8 +6,8 @@
 
 [assembly: AssemblyTitle("Microsoft Azure Data Factory Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Data Factory Resources.")]
-[assembly: AssemblyVersion("4.3.0.0")]
-[assembly: AssemblyFileVersion("4.3.0.0")]
+[assembly: AssemblyVersion("4.3.1.0")]
```
Comment (isra-fel) [inline sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/Properties/AssemblyInfo.cs]: `AssemblyVersion` shouldn't be change unless it's a major version update

File: sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/changelog.md [hunk 3]
```diff
@@ -1,7 +1,11 @@
 # Changelog for the Azure Data Factory V2 .NET SDK
 
 ## Current version
+##
```
Comment (isra-fel) [inline sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/changelog.md]: Remove this extra line please

**Review by isra-fel — CHANGES_REQUESTED** [review]
@mfrintu please see my inline comments and resolve merge conflicts. Thanks

**Comment by isra-fel** [comment]
I got it that version was already changed to 4.4.0 for this release. Please resolve the conflicts then.
You may also close this and open a new PR based on latest master, which I think would be easier.

**Comment by mfrintu** [comment]
Changes were already merged in a different PR.

**Pull request closed without merging**